CDL-Dryad/dryad-app
# Move GMail authentication to the web app

https://github.com/CDL-Dryad/dryad-product-roadmap/issues/1939

Replace the outgoing command-line authentication with a web-based system. The new system performs nearly the same task, but with Google's preferred mechanisms.

To test on a new server:
- go to http://servername/stash/gmail_auth and follow the directions
- after successful authentication, check for the token file one directory *above* the server's root
- verify the token file with `rails journal_email:validate_gmail_connection`
